apps/dao/src/components/Page404/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useRef } from 'react'
+import styled from 'styled-components'
+
+import { breakpoints } from '@/ui/utils/responsive'
+import Box from '@/ui/Box'
+
+export function ErrorVideo() {
+  const videoRef = useRef<HTMLVideoElement>(null)
+
+  useEffect(() => {
+    if (videoRef.current) {
```

Comment:
Either the if should be removed or we need it in the dependencies..?

Reply:
this is some legacy code, I removed it